starter-web-servlet/build.gradle

```diff
@@ -11,11 +11,12 @@ configurations {
 }
 
 dependencies {
-    developmentOnly platform("io.micronaut:micronaut-bom:$micronautVersion")
+    developmentOnly platform("io.micronaut.platform:micronaut-platform:$micronautVersion")
```

Comment:
again, not sure if we should be pulling the platform or just the bom 🤔

Reply:
Platform.